### PR TITLE
Melissa/m641 new project no admin rights bug

### PR DESCRIPTION
**Pull request by mmnoo**
[Ticket](https://trello.com/c/M9bqF5pD/641-admin-but-with-collector-access)

Steps to test:

- Create a new project
- create a new site => expect that you can delete that side (see the delete button)
- create a new MR => expect that you can delete that MR
- expect that you can change project info
- expect that you can make changes to data sharing policy
- expect that you can make changes on the users page
- expect that you can

### CODE REVIEW

**Comment by mmnoo** [comment]
[Admin but with collector access](https://trello.com/c/M9bqF5pD/641-admin-but-with-collector-access)

File: src/components/ProjectCard/ProjectModal.js [hunk 1]
```diff
@@ -81,6 +83,7 @@ const ProjectModal = ({ isOpen, onDismiss, project, addProjectToProjectsPage })
   }
 
   const handleSuccessResponse = (response, languageSuccessMessage) => {
+    refreshCurrentUser() // this ensures the user has the right privaleges to the newly created project
```
Comment (mmnoo) [inline src/components/ProjectCard/ProjectModal.js:86]: Remember how our current user was being initialized way too often? That hid this bug. It wasnt from the upgrade.

Comment (julia-rom) [reply src/components/ProjectCard/ProjectModal.js:86]: ah ok - I suspected it was something else. Nice job!

**Review by julia-rom — APPROVED** [review]
tests pass, new project appears in user's project list now. good work! this one looked tricky.

**Pull request merged**